README.md

```diff
@@ -30,8 +30,12 @@ will look like this:
 lazy val root = (project in file("."))
   .enablePlugins(ProtofetchPlugin)
   .settings(
-    // Use protofetch output as a sbt-protoc source
+    // Use protofetch output as a sbt-protoc source.
+    // This is only necessary if you have proto_out_dir set to
+    // something other than "target/protobuf_external_src"
     Compile / PB.protoSources += protofetchOutputDirectory.value,
```

Comment:
I would consider commenting this line out. So that including it is always a conscious choice

Reply:
I updated the comment.